apiserver/facades/client/modelmanager/modelmanager_test.go

```diff
@@ -148,7 +149,7 @@ func (s *modelManagerSuite) SetUpTest(c *gc.C) {
 						Value: "spam"}}},
 			},
 		},
-		cred: cloud.NewEmptyCredential(),
+		cred: statetesting.CloudCredential(cloud.EmptyAuthType, nil),
```

Comment:
Can we use a statetesting NewEmptyCredential() helper for all these...

Reply:
done